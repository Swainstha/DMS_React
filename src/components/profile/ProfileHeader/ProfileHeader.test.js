import testSnapFunction from '../../../util/testSnapFunction';
import ProfileHeader from './index'
import React from 'react';

testSnapFunction("<ProfileHeader />", "Snapshot test for ProfileHeader ",<ProfileHeader /> )