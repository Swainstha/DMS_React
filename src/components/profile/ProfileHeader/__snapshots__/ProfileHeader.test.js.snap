// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileHeader /> Snapshot test for ProfileHeader  1`] = `ShallowWrapper {}`;
